clients/include/testing_trsm.hpp

```diff
@@ -254,11 +259,12 @@ void testing_trsm(const Arguments& arg)
                     res_2 += std::abs(hXorB_2[j + i * ldb]);
                 }
             }
```

Comment:
If this doesn't exist as a helper I think it should be put there, can be reused in the batch cases.

Reply:
`rocblas_abs` should be used instead of `std::abs` in rocBLAS library code.